mpas_analysis/ocean/ohc_timeseries.py

```diff
@@ -83,7 +114,7 @@ def ohc_timeseries(config, streamMap=None, variableMap=None):
     cp = namelist.getfloat('config_specific_heat_sea_water')
     # [kg/m3]
     rho = namelist.getfloat('config_density0')
-    factor = 1e-22*rho*cp
+    factor = 1e-22
```

Comment:
This magic number is specified in the units of the figure.  It wasn't my doing so I don't feel responsible for fixing it.

However, I did feel that the units in the cache file needed to be correct so I moved the rho*cp into the computation of the cached quantity, which is given in joules, as it should be.